src/geo/ui/infowindow.js

```diff
@@ -307,17 +318,42 @@ var Infowindow = View.extend({
     return false;
   },
 
+  _loadImageHook: function(url) {
+    var $hook = this.$(".js-hook");
+    var $cover = this.$(".js-cover");
+
+    if ($hook) {
+      $hookImage = $("<img />").attr("src", url);
+      $hook.append($hookImage);
+
+      var hookPoints = [[0, 0], [0, 100], [100, 0]];
+
+      $hook.clipPath(hookPoints, {
+        isPercentage: true
+      });
+
+      $hookImage.load(function(){
+        $hookImage.css({
+          marginTop: -$cover.height(),
+          width: $cover.width()
+        });
+      });
+    }
+  },
+
   /**
    *  Attempts to load the cover URL and show it
    */
   _loadCover: function() {
 
-    if (!this._containsCover()) return;
+    if (!this._containsCover()) {
+      return;
+    }
 
     var self = this;
-    var $cover = this.$(".cover");
+
+    var $cover = this.$(".js-cover");
     var $img = $cover.find("img");
-    var $shadow = this.$(".shadow");
     var url = this._getCoverURL();
 
     if ($img.length > 0) {
```

Comment:
If there is already an image defined in the header cover, we should add the class ```CDB-infowindow-media-item``` to that image and we should run ```this._loadImageHook(url);``` before the ```return;```.